test/helpers/thirdPartyRegistry.js

```diff
@@ -81,3 +82,43 @@ export async function getSignature(
     v,
   }
 }
+
+export async function getMessageHash(
```

Comment:
Why do we need to make it to return a promise?

Reply:
I would receive the `chainId` by parameter

Reply:
I'm doing the same as `getSignature` does